src/components/EntityInfo.vue

```diff
@@ -47,18 +47,11 @@
                   <span class="font-13 ml-1">View and Change Business Information</span>
                 </v-btn>
 
-                <v-tooltip top content-class="pending-tooltip" v-if="isPendingDissolution || isNotInCompliance">
+                <v-tooltip top content-class="top-tooltip" transition="fade-transition" v-if="isPendingDissolution">
                   <template v-slot:activator="{ on }">
-                    <span class="pending-alert pr-2" v-on="on">
-                      <v-icon color="orange darken-2">mdi-alert</v-icon>
-                    </span>
-                  </template>
-                  <template v-if="isPendingDissolution">
-                    You cannot view or change business information while the business is pending dissolution.
-                  </template>
-                  <template v-if="isNotInCompliance">
-                    You cannot view or change business information while the business is not in compliance.
+                    <v-icon color="orange darken-2" size="24px" class="pr-2" v-on="on">mdi-alert</v-icon>
                   </template>
```

Comment:
"Not in compliance" is now an alert, so this tooltip is removed for this case (per Tracey).

This tooltip remains for "pending dissolution", except that's not implemented yet (is always false).